app/config/parameters.yml.dist

```diff
@@ -33,6 +35,9 @@ parameters:
     # to enable the comments feature you should pass 'always-active' instead of 'inactive'
     env(COMMENTS_FEATURE): inactive
     env(CREATE_ACCOUNT_FEATURE): inactive
+    env(USE_MAIL_FEATURE): inactive
+
+    env(MAIL_FEATURE_FROM_EMAIL): send@example.com
```

Comment:
The default values should be usable.  `send@example.com` is not.
Probably should be set to ~ (representing null).

Reply:
(might also want to check for it being null before trying to send the email)

Reply:
Changed the value and also added conditional logic in both the controller and the test to make sure the from email value isn't null.